source/triggers/database-triggers.txt

```diff
@@ -137,81 +151,130 @@ Database Triggers have the following configuration options:
-options are available:
-
-.. list-table::
-   :header-rows: 1
-   :widths: 15 30
-
-   * - Field
-     - Description
-
-   * - :guilabel:`Cluster Name`
+Trigger Source Details
+~~~~~~~~~~~~~~~~~~~~~~
 
-     - The name of the MongoDB Service that the Trigger is
-       associated with.
+Within the :guilabel:`Trigger Source Details` section, you first select the 
+:guilabel:`Source Type`, based on the level of granularity you want. Your 
+options are:
 
-   * - :guilabel:`Database Name`
+       - :guilabel:`Collection`, when a change occurs on a specified collection
+       - :guilabel:`Database`, when a change occurs on any collection in a 
+         specified database
+       - :guilabel:`Deployment`, when deployment changes occur on a specified 
+         cluster. If you select the Deployment source type, the following 
+         databases are **not watched** for changes:
 
-     - The MongoDB database that contains the watched
-       collection.
-
-   * - :guilabel:`Collection Name`
-
-     - The name of the collection that the Trigger watches for
-       change events.
-
-   * - :guilabel:`Operation Type`
-
-     - Checkboxes of :ref:`database operation types <database-event-operation-types>` 
-       that cause the Trigger to fire. Select the operation types you want the 
-       trigger to respond to.
-
-       .. warning::
-
-          Update operations executed from MongoDB Compass or the MongoDB Atlas
-          Data Explorer fully replace the previous document. As a result,
-          update operations from these clients will generate **Replace**
-          change events rather than **Update** events.
-
-   * - :guilabel:`Full Document`
-
-     - If enabled, **Update** change events include the latest
-       :manual:`majority-committed </reference/read-concern-majority/>`
-       version of the modified document *after* the change was applied in
-       the ``fullDocument`` field.
+         - The admin databases `admin`, `local`, and `config`
+         - The sync databases `__realm_sync` and `__realm_sync_<app_id>`
+         
+Depending on which source type you are using, the additional options differ. The 
+following table describes these options.
 
-       .. note::
+.. list-table::
+   :header-rows: 1
+   :widths: 15 30
 
-          Regardless of this setting:
-          
-          - **Insert** and **Replace** events always include the
-            ``fullDocument`` field.
-          
-          - **Delete** events never include the ``fullDocument`` field.
+   * - Source Type
+     - Options
 
-   * - :guilabel:`Document Preimage`
+   * - :guilabel:`Collection`
+     - |
+       - :guilabel:`Cluster Name`. The name of the MongoDB cluster that the 
+           Trigger is associated with.
+      
+       - :guilabel:`Database Name`. The MongoDB database that contains the watched
+         collection. Optional if the :guilabel:`Collection Name` is specified.
```

Comment:
```suggestion
         collection. Required if the :guilabel:`Collection Name` is specified.
```

Maybe we should add some copy about how a Deployment trigger is when `database and collection` are not specified. Database trigger is when `collection` is not specified, and Collection Trigger when both are specified.

Reply:
@nathantfrank does this mean that even if the user selects the Collection or Database source type, it will actually be a Deployment type if they leave both `database` and `collection` blank?

Reply:
Ahhh Sorry I misunderstood what this section was doing.
My current thought is we should just remove that sentence.
 ```suggestion
         collection.
```